float32 x
float32 y
float32 theta

float32 response
float32 size

int32 octave
